### ISSUE TITLE
LNDHub QR code gives local address

### ISSUE THREAD

**Issue by wherndon** [mynodebtc/mynode#531]
Raspi 4
MyNode v. 0.2.31

The QR code in the LNDHub app scans in the local address http://mynode.local:3000.  Trying to connect via TOR with the recent BlueWallet update (6.1.1).

**Comment by tehelsper** [comment]
Within the LNDHub web page? Not sure there is much I can do about that. Can you scan from the QR pairing page in myNode?

**Comment by wherndon** [comment]
Yes that worked, thank you!  I missed that the extra bluewalllet +TOR QR codes were generated in the recent update.
